src/components/Dropdown/renderOptions.tsx

```diff
@@ -123,7 +124,7 @@ const RenderOptions = <T extends {}>(props: DropdownProps<T>, ref?: React.Ref<HT
           nodeBeforeItems={!hideGroupByTitle && buildAppendList(item, index)}
           onChange={onChange}
           options={groupedOptions[item]}
-          selected={selected}
+          selected={value}
```

Comment:
@higorquintao we're getting a TS error from this line. It seems to be a missing property type warning. Can you please check?